environments/sre/servicemonitors/thanos-querier-serviceMonitor.yaml

```diff
@@ -0,0 +1,16 @@
+serviceMonitor:
```

Comment:
We need to specifically get this object. So without that top level `serviceMonitor`.